src/components/MoneyRequestReportView/MoneyRequestReportTransactionList.tsx

```diff
@@ -1,34 +1,70 @@
-import React from 'react';
+import React, {useMemo, useState} from 'react';
 import {View} from 'react-native';
+import type {TupleToUnion} from 'type-fest';
+import type {SortOrder} from '@components/Search/types';
+import type {SortableColumnName} from '@components/SelectionList/types';
 import TransactionItemRow from '@components/TransactionItemRow';
 import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useThemeStyles from '@hooks/useThemeStyles';
+import {compareValues} from '@libs/SearchUIUtils';
+import CONST from '@src/CONST';
 import type * as OnyxTypes from '@src/types/onyx';
-import MoneyRequestReportTableHeader from './MoneyRequestReportTableHeader';
+import MoneyRequestReportTableHeader, {columnConfig} from './MoneyRequestReportTableHeader';
 
 type MoneyRequestReportTransactionListProps = {
     /** List of transactions belonging to one report */
     transactions: OnyxTypes.Transaction[];
 };
 
+const columnConfigNames = columnConfig.map(({columnName}) => columnName);
+const unwantedColumnConfigNames = [CONST.SEARCH.TABLE_COLUMNS.RECEIPT, CONST.SEARCH.TABLE_COLUMNS.TYPE, CONST.REPORT.TRANSACTION_LIST.COLUMNS.COMMENTS];
```

Comment:
why `Unwanted` I ma not sure i understand this name in this context. should it be `unsortable` or something like that?